Jenkinsfile

```diff
@@ -0,0 +1,17 @@
+pipeline {
+    agent {label 'jenkins-slave-archita'}
+
+    stages {
+        stage('Build') {
+            steps {
+                bat '''ruby printArrayElements.rb'''
```

Comment:
stage build should get the environment ready and not run the code, you can install gems and other necessary processes here